include/bluetooth/mesh/gen_plvl_srv.rst

```diff
@@ -41,20 +41,20 @@ If the Generic OnOff state is changed to On and the Default Level state is not s
 
 The Power state power up behavior is determined by the On Power Up state of the extended :ref:`bt_mesh_ponoff_srv_readme`:
 
-- :cpp:enumerator:`BT_MESH_ON_POWER_UP_OFF <bt_mesh_ponoff::BT_MESH_ON_POWER_UP_OFF>`: The Power level is set to 0 on power up.
-- :cpp:enumerator:`BT_MESH_ON_POWER_UP_ON <bt_mesh_ponoff::BT_MESH_ON_POWER_UP_ON>`: The Power level is set to Default Level on power up, or the last known   non-zero Power level if the Default level is not set.
-- :cpp:enumerator:`BT_MESH_ON_POWER_UP_RESTORE <bt_mesh_ponoff::BT_MESH_ON_POWER_UP_RESTORE>`: The Power level is set to the last known Power level (zero or otherwise).
+- :c:enumerator:`BT_MESH_ON_POWER_UP_OFF <bt_mesh_ponoff::BT_MESH_ON_POWER_UP_OFF>`: The Power level is set to 0 on power up.
+- :c:enumerator:`BT_MESH_ON_POWER_UP_ON <bt_mesh_ponoff::BT_MESH_ON_POWER_UP_ON>`: The Power level is set to Default Level on power up, or the last known   non-zero Power level if the Default level is not set.
+- :c:enumerator:`BT_MESH_ON_POWER_UP_RESTORE <bt_mesh_ponoff::BT_MESH_ON_POWER_UP_RESTORE>`: The Power level is set to the last known Power level (zero or otherwise).
```

Comment:
Can enumerators drop the explicit linking now?

Reply:
Sorry, missed this; unfortunately enumerators are broken in intersphinx for Sphinx==3.2.1, no matter if using explicit liking or not, but it's already fixed in `3.2.x` branch so when 3.2.2 comes out it should work without explicit linking.